### ISSUE TITLE
jabref firefox extension

### ISSUE THREAD

**Issue by aeneid-prime** [JabRef/JabRef-Browser-Extension#612]
![Screenshot_20240717_132253](https://github.com/user-attachments/assets/72bb505f-333c-4d9c-bbed-2611872d125a)
![Screenshot_20240717_132253](https://github.com/user-attachments/assets/28265b30-f422-430f-8aaa-7e315f7f2e28)

**Comment by LyzardKing** [comment]
Hi,

could you add some information on your system?
What OS are you using, and how you installed firefox and jabref.

With this information we can point you to the solution

**Comment by aeneid-prime** [comment]
I installed firefox using Yast, I downloaded jabref rpm from fosshub.com.   

My  system:

Operating System: openSUSE Tumbleweed 20240708
KDE Plasma Version: 6.1.2
KDE Frameworks Version: 6.3.0
Qt Version: 6.7.2
Kernel Version: 6.9.7-1-default (64-bit)
Graphics Platform: X11
Processors: 4 × Intel® Core™ i5-6300U CPU @ 2.40GHz
Memory: 7.1 GiB of RAM
Graphics Processor: Mesa Intel® HD Graphics 520
Manufacturer: HP
Product Name: HP EliteBook 840 G3

**Comment by LyzardKing** [comment]
Did you try the manual install at https://docs.jabref.org/collect/jabref-browser-extension#manual-installation

That might help.

**Comment by mrtonimendes** [comment]
I have the same problem with Zen Browser (firefox based) installed via flatpak and Jabref installed via flatpak.

Versions:
Zen Browser     io.github.zen_browser.zen     1.0.0-a.28               stable      flathub    system
JabRef                org.jabref.jabref                       5.15                         stable      flathub    system

OS: fresh install of Fedora 40.

**Comment by LyzardKing** [comment]
@mrtonimendes  did you try the manual install?
I assume that Zen Browser has a different folder directory where you should copy the necessary files, but it's not a browser I'm familiar with.

**Comment by divyaranjan1905** [comment]
@LyzardKing I'm on GNU Guix, and installing JabRef from flatpak has also made the extension unusable for me. I'm using it on Librewolf, and the latter is installed from the distro's package manager.

**Comment by hkr260270** [comment]
For me this error occurs on firefox 131.0.3 , jabref browser extension 2.5 and jabref 5.15. In the extensions (about:addon) connection status says : no such native application org.jabref.jabref
I made a normal installation and when it did not work, I did everything manually.

**Comment by LyzardKing** [comment]
@divyaranjan1905 I'm not sure how librewolf deals with native messaging, but I imagine the changes from the manual version would be minimal. The manual instructions should then be fine for librewolf.

@hkr260270 How did you install jabref and firefox? Did you try the manual setup?

**Comment by hkr260270** [comment]
> [@divyaranjan1905](https://github.com/divyaranjan1905) I'm not sure how librewolf deals with native messaging, but I imagine the changes from the manual version would be minimal. The manual instructions should then be fine for librewolf.
> 
> [@hkr260270](https://github.com/hkr260270) How did you install jabref and firefox? Did you try the manual setup?

Yes, I tried the manual setup. Jabref an Firefox have been installed usind standard windows installer an dhave been updated to the actual version.

**Comment by tobiasdiez** [comment]
Closing this issue due to inactivity :zzz: 
Please reopen the issue with additional information if the problem persists.
